tests/ui/lint/dropping_copy_types.stderr

```diff
@@ -22,6 +26,10 @@ LL |     drop(s2);
    |          argument has type `SomeStruct`
    |
    = note: use `let _ = ...` to ignore the expression or result
+help: ignore the value
```

Comment:
Generally we try to make the suggestion explicit about what the it does, this is particularly relevant in IDEs were they might not display the change at all but just the suggestion message.

What about simply reusing the "note" message?
```suggestion
help: use `let _ = ...` to ignore the expression or result
```

Reply:
Done. Thank you.